mbq/metrics/__init__.py

```diff
@@ -119,16 +118,12 @@ def timing(self, metric, value, tags=None):
     def service_check(self, check_name, status, tags=None, message=None):
         # the dogstatsd client doesn't use namespace or constant_tags
         # for service_check but we want to be consistent
-
-        check_name = self._combine_metric(check_name)
-        if self.statsd.namespace:
-            check_name = self.statsd.namespace + '.' + check_name
-
-        tags = self._combine_tags(tags)
-        if self.statsd.constant_tags:
-            tags += self.statsd.constant_tags
```

Comment:
Looks like we were tacking on the tags to the end of the list. We should see if this causes a change in behavior.

Reply:
True it's probably worth testing out if this changes the behavior. I can shell into dev and manually send a metric with duplicate tags and see what happens to those tags in the Datadog dashboard.